mobileversion.css

```diff
@@ -0,0 +1,887 @@
+    border-radius: 8px;
+    opacity: 0.9;
+    width: 100%;
+    text-align: center;
+    transition: 0.3s;
+    -webkit-backdrop-filter: blur(7px);
+    backdrop-filter: blur(7px);
+  }
+}
+
+/* .nav-menu hr {
+  width: 40%;
+  border: 2px solid black;
+  border-radius: 5px;
+  margin-top: 285px;
+  margin-left: 100px;
+  place-items: center;
+} */
+
+.hamburger {
+  cursor: pointer;
+  margin-left: 350px;
+  position: absolute;
+}
+
+.nav-item {
+  margin-top: 50px;
+  margin-bottom: 20px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: blue;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background-color: white;
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background-color: white;
+}
+
+.nav-menu.active {
+  left: 0;
+}
+
+.flex-container {
+  background-image: url("https://user-images.githubusercontent.com/66992901/198218680-01a4bf79-49e0-4ca5-b5a4-363c351466a5.png");
+  display: flex;
+  background-repeat: no-repeat;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  left: 2px;
+  width: 400px;
+  height: 700px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-bottom: 25px;
+  margin-top: -52px;
+}
+
+.item5 {
+  margin-left: -90px;
+}
+
+.item6 {
+  margin-left: -90px;
+}
+
+.time {
+  font-weight: bold;
+  font-size: 15px;
+  padding: 20px;
+}
+
+li i {
+  float: left;
+  color: grey;
+  font-family: 'Poppins', sans-serif;
+  width: 19.99px;
+  height: 19.96px;
+  top: 470px;
+  padding: 1px;
+  margin-top: 2px;
+  margin-left: 20px;
+}
+
+.nav i {
+  width: 40px;
+  margin-left: 400px;
+  color: black;
+}
+
+.mariane {
+  margin-top: 80px;
+  margin-bottom: 10px;
+  margin-left: 70px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  color: #172b4d;
+  display: flex;
+  align-items: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text {
+  color: #344563;
+  margin-left: 70px;
+  margin-top: 10px;
+  font-size: 16px;
+  font-weight: 400;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.lass {
+  margin-left: 56px;
+}
+
+.cap {
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  margin-left: 70px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.containe {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  left: 24px;
+  width: 400px;
+  height: 700px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-bottom: 25px;

```

Comment:
- I think should be better to stick to the figma design as suggested by the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates), here is about the gap between your work cards:
  - Your design:
![gapStudent](https://user-images.githubusercontent.com/97834160/205041168-e78b625c-c2c5-43cb-8972-65f7ed5bfc13.png)


  - Figma design: 
![gap](https://user-images.githubusercontent.com/97834160/205041213-2e4ba5b7-f1c4-4d12-a369-d17661ec0c1b.png)